File: app/src/main/java/com/example/aleksei/reminderapp/model/Note.java
package com.example.aleksei.reminderapp.model;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;
import java.util.Date;

@Entity
public class Note {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private Date noteDate;

    private String noteText;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Date getNoteDate() {
        return noteDate;
    }

    public String getNoteText() {
        return noteText;
    }

    public Note(Date noteDate, String noteText) {
        this.noteDate = noteDate;
        this.noteText = noteText;
    }
}

File: app/src/main/java/com/reminderapp/mvp/data/entity/Note.kt
package com.reminderapp.mvp.data

import androidx.room.Entity
import androidx.room.PrimaryKey
import java.util.Date

@Entity
class Note(val noteDate: Date, var noteText: String) {
    //todo
   @PrimaryKey(autoGenerate = true)
   var id = 0
}